docs/mission-modeling/advanced-the-merlin-interface.mdx

```diff
@@ -5,7 +5,13 @@ Aerie is a software framework for modeling spacecraft. Within Aerie is **Merlin*
 Because mission models are expressed in Java, rather than a custom DSL, Merlin has little to no ability to see the actual Java code comprising a mission model.
 Merlin must instead make inferences about the mission model based on its observable behavior.
 
-Predecessors of Merlin, such as [APGen](https://trs.jpl.nasa.gov/handle/2014/45571) and [SEQGen](https://trs.jpl.nasa.gov/handle/2014/45455), provided a domain-specific language for mission modeling, allowing them to obtain deep, fine-grained information about the composition of a mission model before performing any simulation.
+Merlin is a spiritual successor to the 
+[Blackbird](https://trs.jpl.nasa.gov/handle/2014/52245) planning
+system, which similarly uses Java for activity and resource modeling.
+Blackbird's design shed light on the myriad choices made in designing
+Merlin.
+
+Predecessors of Merlin and Blackbird, such as [APGen](https://trs.jpl.nasa.gov/handle/2014/45571) and [SEQGen](https://trs.jpl.nasa.gov/handle/2014/45455), provided a domain-specific language for mission modeling, allowing them to obtain deep, fine-grained information about the composition of a mission model before performing any simulation.
```

Comment:
The "provided" in this file was changed to "provide" in the other changed file. It would be best if they agreed, although it's a minor nit. (Either "provided" or "provide" is reasonable: "provided" relative to Merlin and "provide" relative to the fact that they're still in use.)